src/satosa/backends/apple.py

```diff
@@ -192,24 +201,49 @@ def response_endpoint(self, context, *args):
         :return:
         """
         backend_state = context.state[self.name]
-        authn_resp = self.client.parse_response(AuthorizationResponse, info=context.request, sformat="dict")
+
+        # Apple sends some user information only via POST in the first request
```

Comment:
is this documented?

Reply:
It's mentioned here (emphasis mine):

> Apple doesn’t receive the user’s full name shared with the system UI. **The raw data is passed directly to your app** from the browser and is not included in the user’s identity token. To help prevent cross-site scripting attacks, validate and sanitize the user-submitted first and last name values before storing on your app servers.

https://developer.apple.com/documentation/sign_in_with_apple/sign_in_with_apple_rest_api/authenticating_users_with_sign_in_with_apple